### PR TITLE
docs: fix links in Markdown files

### PR DESCRIPTION
**Pull request by KhudaDad414**
#### Description
this PR is supposed to fix all the [broken links](https://github.com/asyncapi/converter-go/runs/6143300647?check_suite_focus=true) in markdown files across the repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The contributing process in this project relies on the [GitHub flow](https://gui
    - Updating relevant documents if you add, remove, update props, configuration, or themes
```
Comment (KhudaDad414) [inline CONTRIBUTING.md:16]: @derberg not sure how to handle the above links? 🤔 apparently the files have been removed.

Comment (smoya) [reply CONTRIBUTING.md:16]: We can remove this `CONTRIBUTING.md` and rather refer to the one located in `https://github.com/asyncapi/.github` repository.

Comment (KhudaDad414) [reply CONTRIBUTING.md]: @smoya done.

**Review by smoya — APPROVED** [review]
LGTM!

**Pull request merged**